### PR TITLE
250mb

### PR DESCRIPTION
**Pull request by EliMoshkovich**

### CODE REVIEW

File: .github/workflows/pdp_cicd.yml [hunk 1]
```diff
@@ -26,7 +26,8 @@ jobs:
 
     - name: Pre build PDP tests
       run: |
-        echo "${{ github.event.release.tag_name }}" | cut -d '-' -f 1 > permit_pdp_version
+        #echo "${{ github.event.release.tag_name }}" | cut -d '-' -f 1 > permit_pdp_version
+        echo "1.1.1" | cut -d '-' -f 1 > permit_pdp_version
```
Comment (RazcoDev) [inline .github/workflows/pdp_cicd.yml:30]: Make sure to revert this

File: .github/workflows/pdp_cicd.yml [hunk 2]
```diff
@@ -78,115 +79,115 @@ jobs:
...
+  # build-and-push-pdp:
+  #   needs: pdp-tests
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #   - name: Checkout code
+  #     uses: actions/checkout@v4
+
+  #   - name: Set up QEMU
+  #     uses: docker/setup-qemu-action@v3
+
```
Comment (RazcoDev) [inline .github/workflows/pdp_cicd.yml:135]: this as well

File: Dockerfile [hunk 3]
```diff
@@ -1,174 +1,128 @@
...
 ENV OPAL_SERVER_URL="https://opal.permit.io"
 ENV OPAL_LOG_DIAGNOSE="false"
 ENV OPAL_LOG_TRACEBACK="false"
 ENV OPAL_LOG_MODULE_EXCLUDE_LIST="[]"
 ENV OPAL_INLINE_OPA_ENABLED="true"
 ENV OPAL_INLINE_OPA_LOG_FORMAT="http"
 
-# horizon configuration -----------------------------
-# by default, the backend is at port 8000 on the docker host
-# in prod, you must pass the correct url
```
Comment (RazcoDev) [inline Dockerfile:158]: Why removing the comments?

File: Dockerfile [hunk 4]
```diff
@@ -1,174 +1,128 @@
...
+# Horizon configuration
 ENV PDP_CONTROL_PLANE="https://api.permit.io"
 ENV PDP_API_KEY="MUST BE DEFINED"
 ENV PDP_REMOTE_CONFIG_ENDPOINT="/v2/pdps/me/config"
 ENV PDP_REMOTE_STATE_ENDPOINT="/v2/pdps/me/state"
 ENV PDP_VERSION_FILE_PATH="/app/permit_pdp_version"
 ENV PDP_FACTDB_BINARY_PATH="/app/bin/factdb"
-# This is a default PUBLIC (not secret) key,
```
Comment (RazcoDev) [inline Dockerfile:165]: Why removing the comments?

**Review by RazcoDev — APPROVED** [review]

File: Dockerfile [hunk 5]
```diff
@@ -1,174 +1,128 @@
...
-        touch /bin/factdb ; \
-      fi \
-    fi \
-  ;; \
-  *) \
-    echo "Unknown architecture." ; \
-    exit 1 ; \
-  ;; \
-  esac ; \
+    x86_64) cp /factdb/factstore_server-linux-amd64 /bin/factdb ;; \
```
Comment (omer9564) [inline Dockerfile:33]: You completely removed the allow missing factstore which will break the vanilla pdp

**Pull request merged**